webui/server/server.py

```diff
@@ -17,6 +17,10 @@
 logging.getLogger('werkzeug').setLevel(logging.ERROR)
 logger = logging.getLogger(__name__)
 
+# Edit this to match the serial port name shown in Arduino IDE
+SERIAL_PORT = "/dev/cu.usbmodem83828101"
```

Comment:
Can we change this to /dev/ttyACM0 :)

(being a little selfish since that's what it is for me LOL)

I intend on adding a script to set all this up for you so people won't need to manually edit any files themselves.